packages/atlas/src/components/_channel/ExpandableChannelsList/ExpandableChannelsList.tsx

```diff
@@ -125,26 +125,32 @@ const useChannelsListData = (queryType: ChannelsQueryType, selectedLanguage: str
   const commonOpts: QueryHookOptions = {
     onError: (error) => SentryLogger.error('Failed to fetch channels', 'ExpandableChannelsList', error),
   }
+  const commonWhere = {
+    where: {
+      activeVideosCounter_gt: 4,
+    },
+  }
+
+  const discover = useDiscoverChannels(commonWhere, { ...commonOpts, skip: queryType !== 'discover' })
+  const popular = usePopularChannels(commonWhere, { ...commonOpts, skip: queryType !== 'popular' })
+  const promising = usePromisingChannels(commonWhere, { ...commonOpts, skip: queryType !== 'promising' })
+  // regular channels query needs explicit limit and sorting as it's not defined by Orion
+  const regular = useChannels(
+    {
+      limit: 15,
+      orderBy: ChannelOrderByInput.CreatedAtAsc,
+      where: { ...commonWhere.where, language: { iso_contains: selectedLanguage } },
+    },
+    { ...commonOpts, skip: queryType !== 'regular' }
+  )
```

Comment:
Maybe I wasn't completely clear on the issue, but I don't think we should apply `activeVideosCounter_gt` as default in regular channels query. We're using this on `Channels in your language` and it could result in `No channels found` screen on pretty much most of the languages. Maybe we could omit that, or just set this to just `1`?  @kdembler wdyt?

Reply:
yeah I think since regular query is only used there is safe to make it a `1` we could have it as per case with a prop later if we need to

Reply:
Yeah I think `1` works as a default, we should just override to `4` for the discover channels section. Although we need to test it out on prod and see if 4 actually works well, we may need to adjust

Reply:
I checked these queries(promising/discover/popular) on the graphql playground, and we're getting a lot of results. 

There is another smaller issue, though - results from these queries(promising and discover) are pretty similar, but I don't know what we actually do about it. It's basically expected that the most followed channels will have the most views.